Marlin/Marlin_main.cpp

```diff
@@ -2962,6 +2990,23 @@ inline void gcode_M17() {
   enable_all_steppers();
 }
 
+#ifdef RESUME_FEATURE
+  inline void gcode_M19() {
+    if(code_seen('Z'))
+    {
+      gcode_get_destination(); // For Z
+      prepare_move();
```

Comment:
Ooooh, that makes sense, I think the reason I didn't experience any issues is because when I tested my code I did not have any other movements in queue. Thank you!